Lesson9/merve/styles.css

```diff
@@ -0,0 +1,172 @@
+* {
+    box-sizing: border-box;
+  }
+  body{
+  max-width: 100%;
+  display:flex;
```

Comment:
I think flexbox styles here should not be needed either since all elements inside body element are block level elements
(remove display, flex-direction, justify-content, align-items)